articles/quickstart/backend/rails/interactive.md

```diff
@@ -40,40 +40,41 @@ gem 'jwt'
 bundle install
 ```
 
-## Create a JsonWebToken class {{{ data-action=code data-code="lib/json_web_token.rb" }}}
+## Create an Auth0Client class {{{ data-action=code data-code="app/lib/auth0_client.rb" }}}
```

Comment:
This file needs to be added to the `files` dir: https://github.com/auth0/docs/tree/master/articles/quickstart/backend/rails/files. Note that `app/lib/auth0_client.rb` is the name defined in the YAML frontmatter, e.g. https://github.com/auth0/docs/blob/master/articles/quickstart/backend/rails/files/json_web_token.md?plain=1#L2

Any new files need to be added to the YAML frontmatter of the interactive QS: https://github.com/auth0/docs/blob/master/articles/quickstart/backend/rails/interactive.md?plain=1#L13

Reply:
(and the old file removed)